cli/ops/dispatch_json.rs

```diff
@@ -24,17 +24,19 @@ pub enum JsonOp {
 }
 
 pub fn serialize_result(
-  rust_err_to_json_fn: &'static dyn deno_core::RustErrToJsonFn,
   promise_id: Option<u64>,
   result: JsonResult,
+  get_error_class_fn: deno_core::GetErrorClassFn,
 ) -> Buf {
   let value = match result {
     Ok(v) => json!({ "ok": v, "promiseId": promise_id }),
-    Err(err) => {
-      let serialized_err = (&rust_err_to_json_fn)(&err);
-      let err_value: Value = serde_json::from_slice(&serialized_err).unwrap();
-      json!({ "err": err_value, "promiseId": promise_id })
-    }
+    Err(err) => json!({
+      "err": {
+        "class": (get_error_class_fn)(&err),
```

Comment:
I didn't catch this in the previous PR, but "class" isn't the best name because it's a keyword in JS. Even tho it doesn't cause syntactic errors, I prefer to avoid keywords.

(Apparently TS39 does not care about this, iterators use a method named "return".)

Reply:
> (Apparently TS39 does not care about this, iterators use a method named "return".)

Being able to call a method 'return' is fairly new - ES6 iirc. It does not apply to the `class` keyword tho.
But I was doing `err["class"]` which makes it ok.

But because I'm such a chill dude I've renamed the field to 'className'.